test/regression.js

```diff
@@ -18,11 +19,21 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const width = 960;
 const height = 720;
 
+const FILE_PATTERN = new RegExp('.*');
+let configFileName;
+const DEFAULT_CONFIG = {
+  floatPrecision: 4,
+};
+const CONFIG = configFileName ? readConfigFile(configFileName) : DEFAULT_CONFIG;
+
+const stats = {};
+
 /** @type {PageScreenshotOptions} */
 const screenshotOptions = {
   omitBackground: true,
   clip: { x: 0, y: 0, width, height },
   animations: 'disabled',
+  timeout: 80000,
```

Comment:
Can you explain why you're adding the timeouts?
Given the nature of these tests, I don't think they'd ever be needed to it seems to just be adding noise. 🤔